packages/local-storage-discovery/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@waku/local-storage-discovery",
```

Comment:
nit: not needed at all but maybe `@waku/local-discovery` as it is not important if we use `localStore` or `IndexDB` or whatever